src/main/resources/lang/en.yml

```
@@ -270,6 +270,27 @@ item:
       <arrow> <attr>Drain interval:</attr> %drain_interval%
     waila: "Lava Drainer"
 
+  elevator_1:
+    name: "Elevator I"
+    lore: |-
+      <arrow> Shift to go down, jump to go up.
+      <arrow> Maximum %elevator_range% blocks.
```

```suggestion
      <arrow> <attr>Vertical range:</attr> %elevator_range%
```
Consistency with the other items